File: app/src/main/java/net/chmielowski/fiszki/WordGroup.java
package net.chmielowski.fiszki;

import io.realm.RealmList;
import io.realm.RealmObject;


public class WordGroup extends RealmObject {
    static final String SCORE = "score";
    public RealmList<Word> words;
    public Integer score;

    public WordGroup() {
    }

    WordGroup(RealmList<Word> words) {
        this.words = words;
        this.score = 0;
    }

}

File: app/src/main/java/net/chmielowski/fiszki/WordGroup.kt
package net.chmielowski.fiszki

import io.realm.RealmList
import io.realm.RealmObject

internal open class WordGroup : RealmObject {

    companion object {
        const val SCORE = "score"
    }

    constructor(words: RealmList<Word>) : super() {
        this.words = words
        this.base = words[0]
    }

    constructor() : super()

    lateinit var words: RealmList<Word>
    lateinit var base: Word
    var score = 0
}
